app/models/concerns/exportable.rb

```diff
@@ -172,6 +173,10 @@ def all_external_pledges
     external_pledges.map { |x| "#{x.source} - #{x.amount}" }.join('; ')
   end
 
+  def all_practical_supports
+    practical_supports.map {|ps| "Type: #{ ps.support_type}, Source: #{ps.source}, Confirmed: #{ps.confirmed?}"}.join('; ')
```

Comment:
let's have this follow the pattern in external pledges if that's cool?

```suggestion
    practical_supports.map { |ps| "#{ps.source} - #{ps.support_type} - #{ps.confirmed? ? 'Confirmed' : 'Unconfirmed'}" }.join('; ')
```

Not sure there's a need to define keys inline.